src/main/java/io/antmedia/rest/StreamsSourceRestService.java

```diff
@@ -159,6 +169,30 @@ public Result addSource(Broadcast stream) {
 	}
 
 
+
+	private void addSocialEndpoints(Broadcast streamSource, String socialEndpointIds) {
```

Comment:
Almost same function exists in BroadcastRestService class. This is code duplication. Is it possible to use the same  function in BroadcastRestService? @burak-58

Reply:
Absolutely true. It must be refactored to fix code duplication.